Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -318,13 +318,15 @@ void ImageEditor::set_editor_color_to_color_at_mouse_position(GUI::MouseEvent co
         color = layer->currently_edited_bitmap().get_pixel(position);
     }
 
+    set_appended_status_info(DeprecatedString::formatted("R:{}, G:{}, B:{}, A:{} [{}]", color.red(), color.green(), color.blue(), color.alpha(), color.to_deprecated_string()));
+
     // We picked a transparent pixel, do nothing.
     if (!color.alpha())
         return;
 
-    if (event.button() == GUI::MouseButton::Primary)
+    if (event.buttons() & GUI::MouseButton::Primary)
         set_primary_color(color);
-    else if (event.button() == GUI::MouseButton::Secondary)
+    else if (event.buttons() & GUI::MouseButton::Secondary)
```

Comment:
I don't see the point of this change. We have an else-if, so the cases are still exclusive. If you want to allow both to happen at the same time, remove the else, but this change is unrelated to everything else in the commit. You can add it as another commit on this PR to keep the commit history clean.

Reply:
The reason for this change is on a move event it was not matching the equals conditions and was failing to update while moving the mouse.